Chinese video ad filtering

```diff
@@ -0,0 +1,147 @@
+! Version: 202105021
+! Expires: 2 days
+! Homepage: http://bbs.kafan.cn/thread-1866845-1-1.html
+
+v.youku.com##.control-phonewatch,#right-title-ad-banner
+youku.com##+js(aeld,dblclick,ykplayer.switchFullscreen)
+!v.youku.com##+js(nano-stb)
+||atm.youku.com/
+||cibntv.net/youku/$media
+||valipl.cp31.ott.cibntv.net/$media
+|https://js.ykimg.com/youku/dist/js/lib/ikuAdapterNew.js
+||sealine.youku.com/yk/*callback=adpagesdk$script
+
+iqiyi.com##div[data-adzone],#block-V
+!iqiyi.com##+js(nano-sib)
+iqiyi.com##+js(set, Object.prototype.parseXML, noopFunc)
```

Comment:
These rules are not supported on AdGuard.
Please use `#%#` or `#%#//scriptlet()`.

If you want to get more information, please read [this document](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters).